neuroml/nml/helper_methods.py

```diff
@@ -2294,8 +2298,265 @@ def get_segment_adjacency_list(self):
             except AttributeError:
                 print(f"Warning: Segment: {segment} has no parent")
 
+        self.adjacency_list = child_lists
         return child_lists
 
+    def get_graph(self):
+        """Get a networkx DiGraph of the morphology of the cell with distances
+        between the proximal point of a parent and the point where a child
+        connects to it as the weights of the edges of the graph.
+
+        Please see https://networkx.org/documentation/stable/reference
+        for information on networkx routines that can be used on this graph.
+
+        This method also stores the graph in the `self.cell_graph` attribute
+        for future use.
+
+        :returns: networkx.Graph
+
+        """
+        import networkx as nx
```

Comment:
pep8 doesn't like function level imports

I personally don't care, but IIRC you are flake8ing - doesn't it complain about imports outside the top of the file?

Reply:
Yeh, I wouldn't normally do it, but the helper methods from here are placed in `nml.py` and I don't think there's a way for us to add top level imports into `nml.py` using `generateDS` :thinking:

Reply:
since it is imported in many other places, I might be missing sthing here... can generateDS handle toplevel imports?

Reply:
I think there is: check  `--custom-imports-template`